http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -896,14 +902,17 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
       }
     } yield ()
 
-  private def sendDeleteDisktoWSM(wsmResourceId: WsmControlledResourceId,
-                                  workspaceId: WorkspaceId,
-                                  auth: Authorization
-  )(implicit
-    ev: Ask[F, AppContext]
-  ): F[WsmJobId] =
+  private def deleteDiskInWSM(diskId: DiskId,
+                              wsmResourceId: WsmControlledResourceId,
+                              workspaceId: WorkspaceId,
+                              auth: Authorization
+  )(implicit ev: Ask[F, AppContext]): F[Unit] = {
+    implicit val wsmDeleteDiskDoneCheckable: DoneCheckable[GetDeleteJobResult] = (v: GetDeleteJobResult) =>
```

Comment:
Is this implicit used by `wsmDao.getDeleteDiskJobResult`? Could you add a little comment explaining what it does?

Reply:
I'll pull this out to the top of the class so it can be used across functions